README.md

```diff
@@ -215,6 +226,27 @@ The schema for this type is as follows:
 }
 ```
 
+### Generating an external manifest
```

Comment:
Are there two sections "Generating an external manifest"?

Reply:
yes, thanks, I moved it higher in the file and forgot to remove the first location